mpc_shifting_knapsack/shifting_knapsack.mpc

```diff
@@ -0,0 +1,150 @@
+import math
+
+
+program.use_edabit(True)
+
+
+def show_matrix(m):
+    for i in range(len(m)):
+        print_ln("%s ", m[i].reveal())
+    print_ln("")
+
+
+def knapsack(weights, values, C):
+    '''
+    The recursive param is to choose if solving the problem using the iterative
```

Comment:
Is this comment still relevant? It looks like this is the pure iterative version to me